docs/src/components/GitHubCodeBlock/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+
+const GitHubCodeBlock = (props: propsType) => {
+  const { href, titleSuffixCount, part, lang } = props;
+  const [file, setFile] = useState("");
+  const [title, setTitle] = useState("");
+
+  useEffect(() => {
+    const { user, repo, branch, filePath, title } = parseHref(href, titleSuffixCount);
+    
+    setTitle(title);
+    fetchCodeFromGitHub(
+      `https://raw.githubusercontent.com/${user}/${repo}/${branch}/${filePath}`, setFile, part
+    );
+  }, []);
+
+  return (
+    <div>
+      <h3>{title}</h3>
```

Comment:
We are already using `<h3>` on the pages (e.g. [here](https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Configuration/Connecting-to-the-Database.md)). So I think it will be better to use `<h4>` for this title. Also, the `<h4>` title attracts less attention :)

Reply:
Yeah, it's just a draft. It's not ready to get a review :)